web/zfbc/index.html

```diff
@@ -121,9 +134,15 @@ <h3>Weitere Inhalte demnächst verfügbar</h3>
     <footer>
       <img class="logo-white" src="./../images/logo-white.svg" alt="Logo von Plus" width="36" height="36">
       <div class="centered">
-        <p id="contact">Kontaktieren Sie uns unter <a href="mailto:hello@plus.healthcare" target="_blank" rel="nofollow">hello@plus.healthcare</a> oder <a href="tel:+41786465577" rel="nofollow" class="nobr">+41&#8239;78&#8201;646&#8201;55&#8201;77</a></p>
+        <p id="contact">
+          Kontaktieren Sie uns unter
+          <a href="mailto:hello@plus.healthcare" target="_blank" rel="nofollow">hello@plus.healthcare</a>
+          oder
+          <a href="tel:+41786465577" rel="nofollow" class="nobr">+41&#8239;78&#8201;646&#8201;55&#8201;77</a>
+        </p>
         <p id="copyright">©&nbsp;2019 Plus Healthcare</p>
       </div>
     </footer>
+    <script src='main.js'></script>
```

Comment:
The PR wouldn't actually add a working feature once merged 😢
```suggestion
    <script src='./../main.js'></script>
```

Reply:
oh, you found it before me
2a8bd35 fixes it

Reply:
Some companies have dedicated QA. I see QA not as a separate job but as a responsibility each Dev has for her features 💪 Always be testing your work *from an end user perspective* before pushing 👍